evmcore/dummy_block.go

```diff
@@ -85,6 +94,8 @@ func ToEvmHeader(block *inter.Block, index idx.Block, prevHash hash.Event, rules
 		GasLimit:   math.MaxUint64,
 		GasUsed:    block.GasUsed,
 		BaseFee:    baseFee,
+		Difficulty: difficulty,
+		MixDigest:  common.Hash{}, // TODO provide pseudorandom data?
```

Comment:
A plan to support this is in the making. However, it is beyond the scope of this PR.